doc/install_basic.txt

```diff
@@ -0,0 +1,47 @@
+.. _install_basic:
+
+Basic user install instructions
+===============================
+
+These are the 3 simple ways of obtaining Theano, and additional requirements
+will change depending on the way you choose.
+
+pip
+~~~
+    
+The recommended way to obtain the latest stable version of Theano is from PyPI using
+pip_. Use the [sudo] or [--user] option depending on your admin rights.
```

Comment:
This is not informative enough. Remove this line and explain [sudo] and [--user] (more in detail) in a bulleted list after the command.